examples/shadowdom.js

```diff
@@ -0,0 +1,36 @@
+import { check } from 'k6';
+import { browser } from 'k6/x/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
+export default async function() {
+  const page = browser.newPage();
+  page.setContent("<html><head><style></style></head><body>hello!</body></html>")
+  await page.evaluate(() => {
+    const shadowRoot = document.createElement('div');
+    shadowRoot.id = 'shadow-root';
+    shadowRoot.attachShadow({mode: 'open'});
+    shadowRoot.shadowRoot.innerHTML = '<p id="find">Shadow DOM</p>';
+    document.body.appendChild(shadowRoot);
```

Comment:
It would be good if we could deploy our own service where we can add new html files 🤔 . In the meantime I think this is probably ok. I also wonder if we could add a shadow dom in the `browser.php` file in https://github.com/grafana/test.k6.io.

Not for this PR though, just thought it was worth mentioning.